docker/docker-compose.2004.56.yaml

```
@@ -0,0 +1,19 @@
+version: "3"
+
+services:
+
+  runtime-setup:
+    image: swift-metrics:20.04-5.6
+    build:
+      args:
+        ubuntu_version: "focal"
+        swift_version: "5.6"
+
+  test:
+    image: swift-metrics:20.04-5.6∂
```

```suggestion
    image: swift-metrics:20.04-5.6
```

it looks pretty cool though :)

---

oh lol, thanks :)